test/connector/tcp/mssql/mssql_connector_test.go

```diff
@@ -53,36 +54,46 @@ func RunTests(t *testing.T, queryExec dbQueryExecutor) {
 			cfg.Host = "127.0.0.1"
 		}
 
+		// Execute Query
 		_, err = queryExec(
 			cfg,
 			"",
 		)
 
+		// Test the returned values
```

Comment:
👍 to these section comments

also, similar to my comment above, i think these integration test changes need to be in their own commit.